specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2021-03-01/redisenterprise.json

```diff
@@ -0,0 +1,1643 @@
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true,
+          "description": "Current provisioning status of the database"
+        },
+        "resourceState": {
+          "$ref": "#/definitions/ResourceState",
+          "readOnly": true,
+          "description": "Current resource status of the database"
+        },
+        "clusteringPolicy": {
+          "type": "string",
+          "description": "Clustering policy - default is OSSCluster. Specified at create time.",
+          "enum": [
+            "EnterpriseCluster",
+            "OSSCluster"
+          ],
+          "x-ms-enum": {
+            "name": "ClusteringPolicy",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "evictionPolicy": {
+          "type": "string",
+          "description": "Redis eviction policy - default is VolatileLRU",
+          "enum": [
+            "AllKeysLFU",
+            "AllKeysLRU",
+            "AllKeysRandom",
+            "VolatileLRU",
+            "VolatileLFU",
+            "VolatileTTL",
+            "VolatileRandom",
+            "NoEviction"
+          ],
+          "x-ms-enum": {
+            "name": "EvictionPolicy",
+            "modelAsString": true
+          }
+        },
+        "persistence": {
+          "$ref": "#/definitions/Persistence",
+          "description": "Persistence settings"
+        },
+        "modules": {
+          "type": "array",
+          "description": "Optional set of redis modules to enable in this database - modules can only be added at creation time.",
+          "items": {
+            "$ref": "#/definitions/Module"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      }
+    },
+    "DatabaseList": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Database"
+          },
+          "description": "List of databases"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The URI to fetch the next page of results."
+        }
+      },
+      "description": "The response of a list-all operation."
+    },
+    "Persistence": {
+      "title": "Persistence settings",
+      "description": "Persistence-related configuration for the RedisEnterprise database",
+      "properties": {
+        "aofEnabled": {
+          "type": "boolean",
+          "description": "Sets whether AOF is enabled."
+        },
+        "rdbEnabled": {
+          "type": "boolean",
+          "description": "Sets whether RDB is enabled."
+        },
+        "aofFrequency": {
+          "type": "string",
+          "description": "Sets the frequency at which data is fsynced to disk.",
```

Comment:
Should we say 'flushed' instead of 'fsynced' so we don't have to introduce new terminology?

Reply:
Why is introducing new terminology a problem?  I chose fsynced to be consistent with this doc: https://docs.redislabs.com/latest/rs/concepts/data-access/persistence/

Reply:
Check out the Microsoft style guide. My reason for suggesting was it seems overly technical/implementation-related, and not so 'conversational' or 'human' way of describing things. 

https://docs.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human

Actually, maybe 'saved' or 'written' would be even better. What do you think?

Reply:
Word choice section might be even more helpful. https://docs.microsoft.com/en-us/style-guide/word-choice/

Reply:
Okay thanks, I get it now.  I think either 'saved' or 'written' would work.